notifier.py

```diff
@@ -1,32 +1,63 @@
-import telegram
 import logging
 import random
+import webbrowser
+
+import telegram
+
 
 class NullNotifier:
     def notify(self, properties):
         pass
 
+    @staticmethod
+    def get_instance(config):
+        if not config['enabled']:
+            return NullNotifier()
+        elif 'type' in config and 'browser'.__eq__(config['type'].lower()):
+            return Browsifier(config)
+        else:
+            return Notifier(config)
+
+
 class Notifier(NullNotifier):
     def __init__(self, config):
         logging.info(f"Setting up bot with token {config['token']}")
         self.config = config
         self.bot = telegram.Bot(token=self.config['token'])
-        
 
     def notify(self, properties):
         logging.info(f'Notifying about {len(properties)} properties')
         text = random.choice(self.config['messages'])
-        self.bot.send_message(chat_id=self.config['chat_id'], text=text)
+        for user in self.config['chat_id']:
+            self.bot.send_message(chat_id=user, text=text)
 
         for prop in properties:
             logging.info(f"Notifying about {prop['url']}")
-            self.bot.send_message(chat_id=self.config['chat_id'], 
-                    text=f"[{prop['title']}]({prop['url']})",
-                    parse_mode=telegram.ParseMode.MARKDOWN)
+            for user in self.config['chat_id']:
+                self.bot.send_message(chat_id=user,
+                                      text=f"[{prop['title']}]({prop['url']})",
+                                      parse_mode=telegram.ParseMode.MARKDOWN)
+
+
+class Browsifier(NullNotifier):
```

Comment:
this notifier will open the new properties on browser tabs, by default 5 at once, if you answered 'y' on the console. Otherwise it will just log the links for you.